robotoff/prediction/category/neural/keras_category_classifier_3_0/text_utils.py

```diff
@@ -0,0 +1,1442 @@
+import re
+from collections import defaultdict
+
```

Comment:
Can't we move this to a more general utility ? Isn't there any package doing this (in spacy for example ?).

What about usage of `unicodedata.normalize` ?

Reply:
There is already one in robotoff (utils.text), but as stated above I would like to keep preprocessing code tied to the model to avoid introducing training/inference skew.